pkgs/development/libraries/shibboleth-sp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, autoreconfHook, boost, fcgi, openssl, opensaml-cpp, log4shib, pkgconfig, xercesc, xml-security-c, xml-tooling-c }:
+
+stdenv.mkDerivation rec {
+  name = "shibboleth-sp-${version}";
+  version = "2.6.0";
+
+  src = fetchgit {
+    url = "https://git.shibboleth.net/git/cpp-sp.git";
+    rev = "9ebba5c3a16d03769f436e383e4c4cdaa33f5509";
+    sha256 = "1b5r4nd098lnjwr2g13f04ycqv5fvbrhpwg6fsdk8xy9cigvfzxj";
+  };
+
+  # Needs pkgconfig to find systemd
+  buildInputs = [ boost fcgi openssl opensaml-cpp log4shib pkgconfig xercesc xml-security-c xml-tooling-c ];
+  nativeBuildInputs = [ autoreconfHook ];
+
+  configureFlags = [
+    "--with-xmltooling=${xml-tooling-c}"
+    "--with-saml=${opensaml-cpp}"
+    "--with-fastcgi"
+  ];
+
+  meta = {
+    # apu-1-config was removed from xcode and setting --with-apu1=${aprutil} doesn't
+    # fix the issue, neithe does --with-apu1=${aprutil}/bin/apu-1-config
```

Comment:
I just tried passing in `--without-apxs` in the `configureFlags` on Darwin and it seemed to fix the build. I assume that's what it's inferring on the Linux build anyway, since the apache crap isn't in scope and it can't find it. The main issue is that the configure script is trying to be helpful and look at `/usr/bin`, but in doing so it runs into tooling installed by the Apple SDK and gets confused when it doesn't actually work.